backend/api/v2beta1/swagger/artifacts.swagger.json

```diff
@@ -68,7 +68,7 @@
             "name": "namespace",
             "description": "Required. Namespace of the Artifact's context.",
             "in": "query",
-            "required": false,
```

Comment:
we don't update these files manually, they are auto generated via: https://github.com/opendatahub-io/data-science-pipelines/blob/master/backend/api/Makefile#L26

Reply:
so we would need to do this via the proto file https://github.com/opendatahub-io/data-science-pipelines/blob/master/backend/api/v2beta1/artifacts.proto#L72 

which I'm not sure is possible  https://stackoverflow.com/questions/31801257/why-required-and-optional-is-removed-in-protocol-buffers-3

Reply:
maybe this might be a way: https://github.com/googleapis/googleapis/blob/master/google/api/field_behavior.proto#L33